### DISCUSSION TITLE
Automatically add file type extension to files?

### DISCUSSION THREAD

**Discussion by juzam4** [dbeaver/dbeaver#15253] [Ideas]
This seems like such an easy and obvious feature but when I go to save a file, why does it not automatically add the file extension to the file I'm saving?? For example, I create a new sql script and do file, save as and enter in a filename. The file type is set to *.sql but I have to manually put in the .sql after my filename. If I don't, the file is saved without a file extension and DBeaver doesn't know what to do with it the next time I open the file.

**Comment by LonwoLonwo** [comment]
Hello @juzam4 

What DBeaver version do you use? We fixed that behavior a few versions ago. If your current DBeaver version is not the newest, maybe downloading the new DBeaver version will help you.

**Reply by juzam4** [reply]
Thank you so much, you were spot on. I updated to the latest version and it worked. Thank you!! :)
